daemon/volumes.go

```diff
@@ -52,6 +52,9 @@ func (m *mountPoint) Setup() (string, error) {
 			if !os.IsNotExist(err) {
 				return "", err
 			}
+			if _, err := os.Stat(m.Source); err != nil && os.IsNotExist(err) {
```

Comment:
hah, the worst part is we were already doing this, and this is already wrapped in a stat call... silly me.